wire-ios-data-model/Source/MLS/MLSService.swift

```diff
@@ -1697,7 +1697,9 @@ public final class MLSService: MLSServiceInterface {
                 subconversationType: subconversationType
             )
         } catch DecryptionError.wrongEpoch {
-            await fetchAndRepairGroupIfPossible(with: groupID)
+            Task.detached { [self] in
+                await fetchAndRepairGroupIfPossible(with: groupID)
+            }
```

Comment:
suggestion: leave a comment here explaining why waiting on this will cause a deadlock.

Reply:
added comment